src/D-FSL-1.0.xml

```diff
@@ -0,0 +1,380 @@
+<SPDX name="Deutsche Freie Software Lizenz" identifier="D-FSL-1.0" osi-approved="false">
+  <urls>
+    <url>http://www.dipp.nrw.de/d-fsl/lizenzen/</url>
+    <url>http://www.dipp.nrw.de/d-fsl/index_html/lizenzen/de/D-FSL-1_0_de.txt</url>
+    <url>http://www.dipp.nrw.de/d-fsl/index_html/lizenzen/en/D-FSL-1_0_en.txt</url>
+  </urls>
+  <notes>This license was created for and is backed by the German state. The English translation can be found here:
+         http://www.dipp.nrw.de/d-fsl/index_html/lizenzen/en/D-FSL-1_0_en.txt</notes>
+  <license>
+        <title>
+            <p>Deutsche Freie Software Lizenz</p>
+        </title>
+        <copyright>
+            <p>(c) Ministerium für Wissenschaft und Forschung Nordrhein-Westfalen 2004</p>
+        </copyright>
```

Comment:
I expect this is the copyright for the license text itself, in which case it should not get wrapping `<copyright>` tags but should go into the following `<optional>` section (see #422 for a similar change).  The replaceable text is in the recommended blurb listed at the end of this file, which should be replaced when it's added as a copyright blurb (and should not be replaceable in the license content itself).  I don't think we have explicit markup for “this is the recommended blurb”, but it's a common enough idea that we may want to add something for it later.